services/black/src/ambassador/BlackClient.cs

```diff
@@ -85,8 +93,16 @@ public async Task SetGuestInformationAsync(Guid orderId, GuestInformation guestI
         public async Task<GuestInformation?> GetGuestInformationAsync(Guid orderId)
         {
             ThrowIfNotInitialized();
-            var response = await _senderEndpoint.Request<GuestInformationResponse>(new GuestInformationRequest(orderId)).ConfigureAwait(false);
-            return response?.GuestInformation;
+            var query = _httpClient!.BaseAddress
```

Comment:
Where is the `BaseAddress` defined?
How is it configured to reach the backend?

Reply:
It can be set on the `IHttpClientFactory` while instantiating the client. Maybe I'll add a factory with defaults like the blue ambassador.

Reply:
Hmm maybe I should just inject a `HttpClient`. That way I don't care how and with what base address it is created.

Reply:
yes, maybe an extension method in `IServiceCollection` that registers the typed http client dependency of `BlackClient`, invoked in the startup of the ambassador's host.

I just wanted to ask you if the setup was already in place, or what you though about it

Reply:
Concerning the HttpClient: I looked that up and it should be fine that way. If we host a client application with a generic host, we can use `IServiceCollection.AddHttpClient` and configure the clients that are created with the http client factory.

Concerning the setup: No, I didn't give that a thought. We can discuss that next tuesday.